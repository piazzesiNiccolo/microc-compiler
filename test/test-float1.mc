int main(){
    float a = .2;
    float b = 2.;
    float c = 2e3;
    float d = 2E-3;
    printfloat(a);
    printfloat(b);
    printfloat(c);
    printfloat(d);
}