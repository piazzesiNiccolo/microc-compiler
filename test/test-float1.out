0.200000
2.000000
2000.000000
0.002000
